src/react-components/info-dialog.js

```diff
@@ -13,6 +13,7 @@ class InfoDialog extends Component {
     slack: Symbol("slack"),
     email_submitted: Symbol("email_submitted"),
     invite: Symbol("invite"),
+    safari: Symbol(""),
```

Comment:
Might as well use `Symbol("safari")` here. It makes for nicer debugging at least.